### DISCUSSION TITLE
recording "checked" answers

### DISCUSSION THREAD

**Discussion by Alex-Jordan** [openwebwork/webwork3#124] [General]
In WW2, there was a recent incident where an instructor had a close date of Nov 27. A student came to the set on Nov 30 and "completed" it. The student was seeing green and thought their checked answers (not submitted) were accepted. They apparently did not see the big red "ANSWERS ONLY CHECKED -- ANSWERS NOT RECORDED" message.

Independently, sometime around Dec 1, the instructor changed the close date to Dec 4.

Now that the close date has been changed to later than the original Nov 27, it would be nice if the student's answers from Nov 30 could count for credit. In WW3, is there anything in the works that would allow this?

**Comment by pstaabp** [comment]
To do this (in either ww2 or ww3), it seems like if a close (or reduced scoring) date is changed, we would need to scrape the answer log for checks after the original date.  I guess this is possible.

An issue however if a problem has a fixed number of submissions. Then if the following happens:
- The set closes. 
- A student checks the answer, takes more than the number of submissions and gets the correct answer. 

The student shouldn't get credit so the above theoretical script would also need to count each "check" as a submission. 

Although I think this is possible, I'm not sure it's a great idea.
